gravitee-apim-portal-webui-next/src/components/nav-bar/nav-bar.component.html

```diff
@@ -23,6 +23,11 @@
     @if (!isEmpty(currentUser())) {
       <app-nav-bar-button i18n="@@applications" [path]="['applications']">Applications</app-nav-bar-button>
     }
+    @for (link of customLinks; track link.id) {
+      <a [href]="link.target" mat-stroked-button color="primary" [target]="link.type === 'external' ? '_blank' : ''">
```

Comment:
I would add a class here to make sure that the link's name is on one line. In the preview, it's pushed onto two and it becomes not very readable IMHO.